hoomd/jit/EvaluatorUnionGPU.cuh

```diff
@@ -95,10 +95,10 @@ union_params_t
 
 #ifdef __HIPCC__
 // Storage for shape parameters
-static __device__ union_params_t *d_union_params;
+__device__ union_params_t *d_union_params;
 
 //! Device storage of rcut value
-static __device__ float d_rcut_union;
```

Comment:
I don't know why this change was necessary. Ideally we would move non-static global variables into a file which we know can be compiled and linked exactly once, to avoid ambiguous linkage. De facto this is the case for this header, even though it might create problems if `EvaluatorUnionGPU.cuh` were included from different object files (or fat binaries?). The fact that it works even though the kernel **is** compiled multiple times seems to indicate that NVRTC is not prone to this issue.

Reply:
Right that makes sense. I have no idea what symbols are actually given external linkage by NVRTC, but as you say it seems to work fine.